packages/api-client/src/libs/parse-curl.ts

```diff
@@ -25,6 +26,24 @@ export function parseCurlCommand(curlCommand: string) {
 
   while (arg) {
     if (typeof arg === 'object' && 'op' in arg) {
+      if (arg.op === '&') {
```

Comment:
@antlio what does this do? If we remove it the test for query params still works. Also it seems to be runing the type for query params it should be an array not a record

Reply:
initially to identify query params, i might have added some updates that made this useless. updated it accordingly in #4384 tysm!